SimPEG/data_misfit.py

```diff
@@ -95,23 +119,44 @@ def nP(self):
 
     @property
     def nD(self):
-        """
-        number of data
+        """Number of data.
+
+        Returns
+        -------
+        int
+            Number of data.
         """
         return self.data.nD
 
     @property
     def shape(self):
-        """"""
+        """Shape of the Jacobian.
+
+        The number of data by the number of model parameters.
+
+        Returns
+        -------
+        list of int (n_data, n_param)
```

Comment:
Just a small correction on the type of the returned object:
```suggestion
        tuple of int (n_data, n_param)
```